.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and publish scs-compliance-check Docker image
+
+on:
+  push:
+    branches:
+    - main
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: scs-compliance-check
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
```

Comment:
WARNING - `github.actor` represents the user, that triggered this workflow. it might be itrich, garloff, etc. It might even change when multiple people are working on the same pull request. Is this intended? Or would `github.repository_owner` fit better?

Reply:
I'm not sure. GitHub recommends to use `GITHUB_TOKEN` to authenticate against the GHCR. This would allow everyone with write access to merge a PR and thus writing the image to the registry. What would you recommend? Creating a fine-granular PAT for the SCS organization?

Reply:
I'd propose to leave this at is is for now and see how the build action behaves. We can optimize this once we're getting closer to an official certification start.

Reply:
I have no experience with having github building images and pushing them somewhere and can't judge whether having multiple people pushing images can be a problem or not.

Reply:
We once had the same GitHub Action to create a Docker image for @oss-review-toolkit. It worked fine and was published to the @SovereignCloudStack package list released by @SovereignCloudStack. The `GITHUB_TOKEN` only checks if the user that merges the branch into `main` has the appropriate privileges to write. This is currently the case for every user in our organization and IMHO this is fine for me currently.

Should we ever need to ship a more trustable image, we can still switch this to a reduced group of people.

Let's proceed, get this Action tested and discuss how we want to proceed in the future within the SIG Standardization.